Resources/Prototypes/Entities/Objects/Tools/lantern.yml

```diff
@@ -42,3 +42,11 @@
       radius: 5
       energy: 10
       color: "#FFC458"
+    - type: Flash
```

Comment:
Just as a note, this has only 5 uses before it burns out, which isn't bad necessarily. balance-wise. Just making you aware

Reply:
I'd make it into 10 or something, not really sure how tho.

Reply:
Check out the flash component to see what the datafield is, could be amount:

Reply:
found the datafield, it's "uses:", tried setting it to a different amount, server doesn't run.

Reply:
turns out to just be a blank line at the end, everything works after removing it.